### PR TITLE
Uyuni BV updates

### PR DESCRIPTION
**Pull request by nodeg**
This fixes and aligns the Uyuni BV configuration to what we have in 5.0 with respect to the parameters.

### CODE REVIEW

**Comment by nodeg** [comment]
Damn. There is still something to fix.

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf [hunk 1]
```diff
@@ -164,7 +164,7 @@ module "base_s390" {
 }
 
 module "server_containerized" {
-  source             = "./modules/server"
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:167]: oops

probably my bad...

**Review by Bischoff — APPROVED** [review]
lgtm, nice catches

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf (hunk 1 above)
Comment (nodeg) [reply terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:167]: No worries.

**Comment by nodeg** [comment]
Alright, the CI tests pass now. I forgot something.

**Review by vandabarata — APPROVED** [review]

**Pull request merged**